R/fill.R

```diff
@@ -19,8 +19,94 @@ NULL
 #'   or "updown" (first up and then down).
 #' @export
 #' @examples
-#' df <- data.frame(Month = 1:12, Year = c(2000, rep(NA, 11)))
-#' df %>% fill(Year)
+#' # Here we have some spreadsheet data where the analyst has only updated year
+#' # when the year has changed, leading to a consistent placement of year at the top
+#' # followed by trailing NAs up to the next year
+#'
+#' sales <- tibble(quarter = rep(c("Q1", "Q2", "Q3", "Q4"), 4),
```

Comment:
Can you please indent long lines using our standard rules? https://style.tidyverse.org/syntax.html#long-lines

You might also consider using a `tribble()` to make the placement of the missing values more clear.

Reply:
Will do, thanks for feedback!